### PR TITLE
Fix type annotations originating in property initializers being emitted in constructor

### LINKED ISSUE #31792: Emitting type parameters (regression)
**TypeScript Version:**  master

**Search Terms:**

**Code**

```ts
export class MyMap<K, V> {
	constructor(
		private readonly Map_: typeof Map = Map
	) {
	}
	private readonly store = new this.Map_<K, V>();
}
```

**Expected behavior:**

```js
export class MyMap {
    constructor(Map_ = Map) {
        this.Map_ = Map_;
        this.store = new this.Map_();
    }
}
```

**Actual behavior:**

```js
export class MyMap {
    constructor(Map_ = Map) {
        this.Map_ = Map_;
        this.store = new this.Map_<K, V>();
    }
}
```

**Playground Link:** 

**Related Issues:**

### PR DESCRIPTION
**Pull request by andrewbranch**
Fixes #31792

### CODE REVIEW

File: src/compiler/transformers/ts.ts [hunk 1]
```diff
@@ -916,7 +916,7 @@ namespace ts {
                 );
 
                 const parameters = transformConstructorParameters(constructor);
-                const body = transformConstructorBody(node.members, constructor, parametersWithPropertyAssignments);
+                const body = transformConstructorBody(existingMembers, constructor, parametersWithPropertyAssignments);
```
Comment (andrewbranch) [inline src/compiler/transformers/ts.ts]: TL;DR: `node.members` are the original, untransformed members; `existingMembers` have been transformed through `visitTypeScript`.

**Comment by ahejlsberg** [comment]
@typescript-bot test this

**Comment by mheiber** [comment]
PR that broke this was https://github.com/microsoft/TypeScript/pull/30467, reverted in https://github.com/microsoft/TypeScript/pull/31807

**Comment by andrewbranch** [comment]
Closing as #30467 was reverted

**Pull request closed without merging**